### DISCUSSION TITLE
WSJTX

### DISCUSSION THREAD

**Discussion by frrobert2** [mbridak/not1mm#96] [Q&A] [answered]
Does not1mm have support for WSJTX?  I tried it the other day and I could not get it to work.  I was just checking if it was me or it is not supported?

Thanks.

**Comment by mbridak** [comment, marked as answer]
Hello Sir, Currently it does not. 
I had written a WSJT-X UDP packet watcher for my stand alone Field Day logging program. 
I suppose it would not be very hard to rip it out and replant it here. I don't do much FT8/FT4 or data modes in general. So I hadn't gotten to it yet. 
I also need to write an XMLRPC interface to FLDigi for RTTY mode support.

Thanks for trying the program out.
Cheers, Mike K6GTE

**Reply by frrobert2** [reply]
Thanks for the information and the quick response.

**Reply by mbridak** [reply]
I put a wsjt-x QSO listener in awhile ago. Forgot to update this thread.

**Comment by py4xx** [comment]
Hi Mike... 
Is there already a way for not1mm to log WSJT-X QSOs?
I put port 7362 on not1mm and wsjt-x, but nothing happened when logging into wsjt-x.


![not1mm_wsjtx_2](https://github.com/user-attachments/assets/916fd248-7f7a-4d9e-bc74-1b4f7b4908f1)
![not1mm_wsjtx](https://github.com/user-attachments/assets/0ffcaa9f-787c-4e15-a1dd-1dd67669328e)

Am I doing something wrong?
TNX.
Carlos-PY4XX

**Comment by mbridak** [comment]
Not1MM listens for WSJT-X UDP ADIF traffic on the Multicast address 224.0.0.1:2237.

Ignore the N1MM configuration tab.

![image](https://github.com/user-attachments/assets/470edb9f-9cc7-4880-9152-236f6db97fb5)

**Comment by py4xx** [comment]
![wsjtx_3](https://github.com/user-attachments/assets/f5ee826b-65a1-432e-b3f9-9ae19c0398ef)

I don't know what I'm doing wrong... I configured it as directed, but it still doesn't send to not1mm...

**Comment by mbridak** [comment]
It may not be anything you are doing. What contest do you have active when you're trying?

**Comment by mbridak** [comment]
If you launch not1mm from a terminal.
then launch wsjtx.

in the terminal you launched not1mm from you should see some status messages dumped to the terminal like:
```text
heartbeat: sv:2 p:0 u:WSJT-X: ms:3 av:2.7.0-rc8
Status: sv:2 p:1 u:WSJT-X df:0 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:0 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:0 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:0 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:0 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:0 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:0 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:14074000 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:145000000 m:FT8 dxc:
Status: sv:2 p:1 u:WSJT-X df:14074000 m:FT8 dxc:
heartbeat: sv:2 p:0 u:WSJT-X: ms:3 av:2.7.0-rc8
```

I don't think it works in the general logger, that's why I was asking which contest you have running.

**Comment by mbridak** [comment]
I added the ft8 listener to the general logger.
Just in case that was the issue.

**Reply by py4xx** [reply]
Hi Mike... 

 Now it's working... 
I put General Logging...

TNX

**Comment by mbridak** [comment]
Okay good. I still have to make some changes for it to make more sense.

**Comment by py4xx** [comment]
Hi Mike... 
This past weekend, I participated in LABRE DIGI... Our not1mm worked perfectly well, capturing the WSJT-X logs.
I did a test, putting WSJT + MSHV together to see if I could rent not1mm with both, but I couldn't. MSHV did not log into not1mm in any way. Maybe, because not1mm can only log in from one??? I don't know. Why MSHV? I think he listens a lot more than WSJT. Several times, MSHV decoded and WSJT did not. So I made contact, but I had to log in manually to not1mm. Unfortunately, I'm having problems with MSHV, because when I close the program it doesn't save the settings, and when I open it, I have to configure everything again... Ugh!!! But not1mm was big... 
73's.

**Comment by mbridak** [comment]
Until this very moment I was unaware of MSHV. Unless it sends it's ADIF UDP packet in the exact format as WSJT, it will be ignored.
I'll download it and see what I can find out about what it is doing.
